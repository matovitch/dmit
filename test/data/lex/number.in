1
042
1.
.1
0.0
23.1415
1.14e0
.11e+71