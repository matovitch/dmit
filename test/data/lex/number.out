{"tokens" :["START_OF_INPUT","INTEGER","WHITESPACE","INTEGER","WHITESPACE","DECIMAL","WHITESPACE","DECIMAL","WHITESPACE","DECIMAL","WHITESPACE","DECIMAL","WHITESPACE","DECIMAL","WHITESPACE","DECIMAL","END_OF_INPUT"],"offsets":[38,37,36,33,32,30,29,27,26,23,22,15,14,8,7,0,0]}